helpers/focus.js

```diff
@@ -163,9 +163,9 @@ export function getPreviousFocusableAncestor(node, includeHidden, includeTabbabl
 	return null;
 }
 
-export function isFocusable(node, includeHidden, includeTabbablesOnly) {
+export function isFocusable(node, includeHidden, includeTabbablesOnly, includeDisabled) {
```

Comment:
In retrospect, we probably should have done this with an `options` parameter with smart defaults. Then you could specify one without needing to specify the rest.